app/controllers/concerns/sufia/works_controller_behavior.rb

```diff
@@ -23,6 +23,35 @@ def edit
       super
     end
 
+    def actor
+      @actor ||= begin
+                   inner_actor = CurationConcerns::CurationConcern.actor(curation_concern, current_user, attributes_for_actor)
```

Comment:
Did you mean to use the new `#actor_class` method here instead of `CurationConcerns::CurationConcern` directly?